temporal-sdk/src/main/java/io/temporal/workflow/NexusOperationOptions.java

```diff
@@ -45,6 +46,19 @@ public NexusOperationOptions.Builder setScheduleToCloseTimeout(
       return this;
     }
 
+    /**
+     * Sets the cancellation type for the Nexus operation. Defaults to WAIT_COMPLETED. Note:
+     * EXPERIMENTAL
```

Comment:
Oh sorry I missed this, for the Java SDK we have an annotation to mark APIs as experimental you can add @Experimental to the API and type 





temporal-sdk/src/main/java/io/temporal/common/Experimental.java